schema/data/test_setup/external_database_setup.sql

```diff
@@ -26,3 +26,4 @@ values
 
 grant usage on schema swrs to foreign_user;
 grant select on all tables in schema swrs to foreign_user;
+grant insert on all tables in schema swrs to foreign_user;
```

Comment:
is this grant necessary? We aren't inserting anything in the swrs schema are we?

Reply:
We are in the test, to test the update of a SWRS name